### PR TITLE
Small changes to the CONTRIBUTING readme

### PR DESCRIPTION
**Pull request by pierotibou**
### Description

Tiny updates following some back and forth today. 
Also removed the php 5 examples as we don't support it anymore

### Readiness checklist
- [ ] (only for Members) Changelog has been added to the release document.
- [ ] Tests added for this feature/bug.

### Reviewer checklist
- [ ] Appropriate labels assigned.
- [ ] Milestone is set.
- [ ] Changelog has been added to the release document. For community contributors the reviewer is in charge of this task.

### CODE REVIEW

**Review by realFlowControl — APPROVED** [review]
Hey there,
I really like that this is now in a chronological order 🎉 
nit(non-blocking): We could add PHP 8.1 and 8.2 `docker-composer` commands as well
LGTM!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -99,45 +111,42 @@ Then you can run tests:
...
     $ make test_web
 
+    # Run C Tests (the ones in tests/ext)
+    $ make test_c
+
 In order to run the `phpt` tests for the php extension:
 
 ```bash
 $ composer test-ext
 ```
```
Comment (bwoebi) [inline CONTRIBUTING.md:142]:
```suggestion
```
This can be removed then, when we recommend `make test_c` above.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -99,45 +111,42 @@ Then you can run tests:
...
 
-### PHP linting
-
-The PHP tracer conforms to the [PSR-2 coding style guide](https://www.php-fig.org/psr/psr-2/). The code style is checked with [PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) which can be invoked with the following command:
+#### Treardown the environment
```
Comment (bwoebi) [inline CONTRIBUTING.md:144]:
```suggestion
#### Teardown the environment
```
typo

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -99,45 +111,42 @@ Then you can run tests:
...
+    # Run C Tests (the ones in tests/ext)
+    $ make test_c
+
 In order to run the `phpt` tests for the php extension:
 
 ```bash
 $ composer test-ext
 ```
 
-### PHP linting
```
Comment (bwoebi) [inline CONTRIBUTING.md:129]: This section is still relevant, we just removed the C code style, not the PHP code style.

**Pull request merged**